modules/libcom/RTEMS/posix/rtems_init.c

```diff
@@ -1023,7 +1026,9 @@ POSIX_Init ( void *argument __attribute__((unused)))
     on_exit(default_network_on_exit, NULL);
 
     /* Let other tasks run to complete background work */
-    default_network_set_self_prio(RTEMS_MAXIMUM_PRIORITY - 1U);
+    sc = rtems_task_set_priority (RTEMS_SELF, RTEMS_MAXIMUM_PRIORITY/2, &old);
+    assert(sc == RTEMS_SUCCESSFUL);
+    printf("Priority changed from %d -> %d\n", old, RTEMS_MAXIMUM_PRIORITY/2);
```

Comment:
The comment says:
```
/* Let other tasks run to complete background work */
```
On SMP anything able to run would be running. It is not clear to me what this code is doing?